packages/flutter/lib/fix_data/fix_services.yaml

```diff
@@ -18,6 +18,31 @@
 # * Fixes in this file are from the Services library. *
 version: 1
 transforms:
+  # Changes made in https://github.com/flutter/flutter/pull/82594 and https://github.com/flutter/flutter/pull/119187
+  - title: "Migrate to 'routeInformationUpdated'"
+    date: 2023-01-30
+    element:
+      uris: [ 'services.dart' ]
+      method: 'routeUpdated'
+      inClass: 'SystemNavigator'
+    changes:
+      - kind: 'removeParameter'
+        name: 'previousRouteName'
+      - kind: 'removeParameter'
+        name: 'routeName'
+      - kind: 'addParameter'
+        index: 0
+        name: 'location'
+        style: optional_named
+        argumentValue:
+          expression: '{% location %}'
```

Comment:
This part doesn't seem to work, the parameter doesn't get added.

Reply:
I think the `requiredIf:  key is necessary since it is an optional parameter

Reply:
Tried that in https://github.com/flutter/flutter/pull/119187/commits/a18622d007170d030188e1b5171cde38c465c744, it didn't work :/